docs/scripts/formattedTSDemos.js

```diff
@@ -102,7 +102,7 @@ async function transpileFile(tsxPath, program) {
 
     const propTypesAST = typescriptToProptypes.parseFromProgram(tsxPath, program, {
       shouldResolveObject: ({ name }) => {
-        if (name === 'classes') {
+        if (name === 'classes' || name === 'ownerState') {
```

Comment:
There were lint issues while generating the `ownerState` in the JavaScript files. Also, it cluttered a lot the demo, so I am skipping the `ownerState` here, so that it shows only as object in the `propTypes`.